### PR TITLE
[ROSTESTS] Fix some tests that fail on WHS testbot

### PR DESCRIPTION
**Pull request by tkreuzer**
## Purpose

More test fixes

## TODO

- [x] Run Test WHS
-- Build: https://build.reactos.org/#/builders/2/builds/2863
-- Testman: https://reactos.org/testman/compare.php?ids=86860,86862,86865,86866,86869
- [x] Run Test KVM
-- Build: https://build.reactos.org/#/builders/9/builds/33792
-- Testman: https://reactos.org/testman/compare.php?ids=86848,86852,86854,86856,86871

### CODE REVIEW

**Review by learn-more — APPROVED** [review]

File: modules/rostests/apitests/acpi/Bus_PDO_QueryResourceRequirements.c [hunk 1]
```diff
@@ -501,7 +501,7 @@ START_TEST(Bus_PDO_QueryResourceRequirements)
     ok(ReqList->InterfaceType == Internal, "InterfaceType = %u\n", ReqList->InterfaceType);
     ok(ReqList->BusNumber == 0, "BusNumber = %lu\n", ReqList->BusNumber);
     ok(ReqList->SlotNumber == 0, "SlotNumber = %lu\n", ReqList->SlotNumber);
-    ok(ReqList->AlternativeLists == 2, "AlternativeLists = %lu\n", ReqList->AlternativeLists);
+    ok(ReqList->AlternativeLists == 1, "AlternativeLists = %lu\n", ReqList->AlternativeLists);
```
Comment (ThFabba) [inline modules/rostests/apitests/acpi/Bus_PDO_QueryResourceRequirements.c:504]: This is not right. This test is a true unit test, it runs against ROS's implementation of this function even on Windows.
So we could either find a way to skip it on Windows, or mark these two failing tests as todo.

**Review by ericsomto — CHANGES_REQUESTED** [review]
Test program before merge

File: modules/rostests/apitests/ntdll/CMakeLists.txt [hunk 2]
```diff
@@ -139,7 +139,7 @@ add_rostests_file(TARGET ntdll_apitest SUBDIR testdata)
 # These are empty files
 add_rostests_file(FILE "${CMAKE_CURRENT_SOURCE_DIR}/ntdll_apitest.exe.local" SUBDIR testdata)
 add_rostests_file(FILE "${CMAKE_CURRENT_SOURCE_DIR}/shell32.dll" SUBDIR testdata)
-add_rostests_file(FILE "${CMAKE_CURRENT_SOURCE_DIR}/test.dll" SUBDIR testdata)
+# add_rostests_file(FILE "${CMAKE_CURRENT_SOURCE_DIR}/test.dll" SUBDIR testdata)
```
Comment (HBelusca) [inline modules/rostests/apitests/ntdll/CMakeLists.txt:142]: Commit "[NTDLL_APITEST] Try to fix test for NtMaßViewOfSection" --> should read "NtMapViewOfSection"

File: modules/rostests/apitests/acpi/Bus_PDO_QueryResourceRequirements.c (hunk 1 above)
Comment (tkreuzer) [reply modules/rostests/apitests/acpi/Bus_PDO_QueryResourceRequirements.c]: Removed that part

File: modules/rostests/apitests/ntdll/CMakeLists.txt (hunk 2 above)
Comment (tkreuzer) [reply modules/rostests/apitests/ntdll/CMakeLists.txt:143]: Fixed

**Review by JoachimHenze — APPROVED** [review]

**Review by GeoB99 — APPROVED** [review]

**Review by learn-more — APPROVED** [review]

File: modules/rostests/apitests/ntdll/NtMapViewOfSection.c [hunk 3]
```diff
@@ -1818,8 +1818,8 @@ Test_RawSize(ULONG TestNumber)
             ok_hex(ImageFile->zdata_header.SizeOfRawData, RawSizeImageFile.zdata_header.SizeOfRawData);
             ok_hex(ImageFile->zdata_header.PointerToRawData, 0);
 
-            /* PointerToRawData = 0 resets SizeOfRawData to 0, CORE-18797 */
-            ok_hex(ImageFile->bss_header.SizeOfRawData, 0);
+            /* .bss section is unmodified */
```
Comment (HBelusca) [inline modules/rostests/apitests/ntdll/NtMapViewOfSection.c:1821]: Any explanation then why https://git.reactos.org/?p=reactos.git;a=commit;h=9293b975ef6 was a priori needed?

**Review by HBelusca — APPROVED** [review]

File: modules/rostests/apitests/ntdll/NtMapViewOfSection.c (hunk 3 above)
Comment (tkreuzer) [reply modules/rostests/apitests/ntdll/NtMapViewOfSection.c:1821]: I don't know, but the test was obviously broken.

**Pull request merged**